examples/node_pool/main.tf

```diff
@@ -57,16 +58,17 @@ module "gke" {
       max_count         = 2
       disk_size_gb      = 30
       disk_type         = "pd-standard"
-      accelerator_count = 1
-      accelerator_type  = "nvidia-tesla-p4"
       image_type        = "COS"
       auto_repair       = false
       service_account   = "${var.compute_engine_service_account}"
     },
   ]
 
   node_pools_oauth_scopes = {
-    all = []
+    all = [
```

Comment:
These look like unrelated changes. Can you rebase on master?

Reply:
i can, but the example keeps recreating `node_pools` on every converge without this being set.